tests/phpunit/tests/option/option.php

```diff
@@ -361,14 +361,114 @@ public function data_option_autoloading() {
 		return array(
 			array( 'autoload_yes', 'yes', 'yes' ),
 			array( 'autoload_true', true, 'yes' ),
-			array( 'autoload_string', 'foo', 'yes' ),
-			array( 'autoload_int', 123456, 'yes' ),
-			array( 'autoload_array', array(), 'yes' ),
+			array( 'autoload_string', 'foo', 'default-yes' ),
+			array( 'autoload_int', 123456, 'default-yes' ),
+			array( 'autoload_array', array(), 'default-yes' ),
```

Comment:
Why are we changing these existing data sets? I don't think we should do that. For any new checks, we should add new datasets to this method.